lib/modules/dosomething/dosomething_signup/includes/SignupsController.php

```diff
@@ -2,6 +2,8 @@
 
 class SignupsController extends EntityAPIController {
 
+  protected $ignore_global_user = FALSE;
```

Comment:
@weerd hmm, I think I'd rather we update the method to accept the user as a param than have this work around